marlowe/test/Spec/Marlowe/Marlowe.hs

```diff
@@ -67,7 +67,7 @@ getSignaturesTest :: IO ()
 getSignaturesTest = do
     let alicePk = walletPubKey alice
     let bobPk = walletPubKey bob
-    let deposit pk = IDeposit (AccountId 0 alicePk) pk (Ada.adaOf 256)
+    let deposit pk = IDeposit (AccountId 0 alicePk) pk "" "" 256_000000
```

Comment:
nitpick `256_000_000` (also same below)

Reply:
I just wanted to separate whole Ada from decimal Lovelaces.